144. Binary Tree Preorder Traversal.java

```diff
@@ -0,0 +1,24 @@
+/**
+ * Definition for a binary tree node.
+ * public class TreeNode {
+ *     int val;
+ *     TreeNode left;
+ *     TreeNode right;
+ *     TreeNode(int x) { val = x; }
+ * }
+ */
+public class Solution {
+    List<Integer> lst = new ArrayList<Integer>();
+    public List<Integer> preorderTraversal(TreeNode root) {
```

Comment:
It's probably better to move the `List<Integer> lst` in here, and pass it as an argument to `preorder`, but its ok.